### ISSUE TITLE
AdminToolkit_Mod_Players dosnt work

### ISSUE THREAD

**Issue by Warsheep-GER** [ole1986/a3-adminhelper#1]
i tryed to include AdminToolkit_Mod_Players  inside my config.cpp(mission) but it dosnt work ,
i found out thats in the clientside modaddon it only gives 

```
Line 121:       if(!(isNil {missionNamespace getVariable "AdminToolkit_Mod_Vehicles"}) ) then 
Line 123:           _list = [] call AdminToolkit_Mod_Vehicles;
Line 143:       if(!(isNil {missionNamespace getVariable "AdminToolkit_Mod_Weapons"}) ) then
Line 145:             _list = [] call AdminToolkit_Mod_Weapons;
Line 170:       if(!(isNil {missionNamespace getVariable "AdminToolkit_Mod_Custom"})) then 
Line 172:           _list = [] call AdminToolkit_Mod_Custom;
Line 190:       if(!(isNil {missionNamespace getVariable "AdminToolkit_Mod_Other"})) then 
Line 192:           _list = [] call AdminToolkit_Mod_Other;
Line 213:       if(!(isNil {missionNamespace getVariable "AdminToolkit_Mod_Items"})) then 
Line 215:           _list = [] call AdminToolkit_Mod_Items;
```

so its missing it for AdminToolkit_Mod_Players  (possible this is the issu)

**Comment by ole1986** [comment]
Hi, thank you for reporting.

Actually it was supposed to work like this as the play are usually "always" used.

Is there a reason to specifically use the players tab instead of any other?

**Comment by Warsheep-GER** [comment]
ya i have a lot of to used on the players section. 
like kill/heal/hunger/thirst/  give/removed=moeny/respect cuffing(freez)  ec, ec, 
not all but the most of them are usefull in some cases. 
ill thinked about a 2. playersbutton. but ill think 6 buttons are not enugh for me and all my stuff i will do with this menu  :D 

ill realy like ur atk , its easy to configurate. just need to search howto make bisign bikeys , then i dont need to change it in the missionfiles ;) 
and because i love ur tool i will shared all my stuff with u

**Comment by ole1986** [comment]
@Warsheep-GER 

I would be very happy when you contribute to this project - You can simply fork, branch and do pull request to this and we are good to go...

There is no need to sign - Simply remove the *.bisign and the bikeys (on server) FOR TESTING PURPOSES and and I will do the rest, once you requested for a merge...

**Comment by Warsheep-GER** [comment]
i removed the admintoolkit.pbo.admintoolkit.bisign on client and the admintoolkit.bikey on server and get kicked for missing signatur key

**Comment by ole1986** [comment]
Mhm alternatively you can disable the verification from server config
Or more Additionally...!? you maybe need do both

This is the line for your server config:
verifySignatures = 1;

**Comment by Warsheep-GER** [comment]
will be fixed with the new drop down system #4
